TitleBarCustomization/explainer.md

```diff
@@ -11,8 +11,8 @@ This document is intended as a starting point for engaging the community and sta
 
 ## Table of Contents
  - [Introduction](#introduction)
- - [Examples of title bar customization on desktop apps](#examples-of-title-bar-customization-on-desktop-apps)
- - [Problem to solve: PWA Title bar area is system reserved](#problem-to-solve-pwa-title-bar-area-is-system-reserved)
+ - [Examples of desktop apps customizing the title bar area](#examples-of-title-bar-customization-on-desktop-apps)
```

Comment:
Link is broken. should be `[](#examples-of-desktop-apps-customizing-the-title-bar-area)` to match new header text. Same applies to all the other changed headings

Reply:
fixed.